### PR TITLE
fix(notice,chip,popover): setting closable prop to false will remove dismissible prop

### PR DESCRIPTION
**Pull request by anveshmekala**
**Related Issue:** #5043 

## Summary

This fix will make sure the `dismissible` prop will be removed if `closable=false` and vice-versa.

### CODE REVIEW

File: src/components/popover/popover.tsx [hunk 1]
```diff
@@ -77,9 +77,18 @@ export class Popover implements FloatingUIComponent, OpenCloseComponent {
    */
   @Prop({ reflect: true }) dismissible = false;
 
+  handleDismissible(value: boolean): void {
```
Comment (driskull) [inline src/components/popover/popover.tsx:80]: does this need a watcher?

**Review by driskull — CHANGES_REQUESTED** [review]
I think there's a watch missing. otherwise looks good 👍

File: src/components/popover/popover.tsx (hunk 1 above)
Comment (anveshmekala) [reply src/components/popover/popover.tsx:80]: This needs the watcher to update `closable` value .

Comment (driskull) [reply src/components/popover/popover.tsx:80]: Is it missing?

Comment (anveshmekala) [reply src/components/popover/popover.tsx:81]: Ahh i see it now, missed the decorator.

Comment (anveshmekala) [reply src/components/popover/popover.tsx:81]: Will add an e2e test to make sure both props are in sync.

**Review by driskull — APPROVED** [review]
👏 🌈

**Pull request merged**